src/invidious/videos/parser.cr

```diff
@@ -53,9 +53,9 @@ end
 def extract_video_info(video_id : String)
   # Init client config for the API
   client_config = YoutubeAPI::ClientConfig.new
-  # Use the WEB_CREATOR when po_token is configured because it fully only works on this client
+  # Use the WEB when po_token is configured
   if CONFIG.po_token
-    client_config.client_type = YoutubeAPI::ClientType::WebCreator
+    client_config.client_type = YoutubeAPI::ClientType::Web
   end
```

Comment:
The default client type is already WEB so this section can be removed

Reply:
doesn't this allow to explicitly say that we use WEB? I was thinking the same, though. I don't know.

Reply:
I mean the logic will also default to the WEB client when the conditional fails. Its moot to include that.

Might be easier to just do 

```crystal
  if CONFIG.po_token
    client_config.client_type = YoutubeAPI::ClientType::Web
  else
    client_config.client_type = YoutubeAPI::ClientType::AndroidTestSuite
  end
```

As so we only have a single request. Though I guess the android client also doesn't return all of the data we need